docs/reference/esql/functions/search-functions.asciidoc

```diff
@@ -6,11 +6,13 @@
 ++++
 
 Full text functions are used to search for text in fields.
-<<analysis, Text analysiss>> is used to analyze the query before it is searched.
+<<analysis, Text analysis>> is used to analyze the query before it is searched.
 
 Full text functions can be used to match <<esql-multivalued-fields,multivalued fields>>.
 A multivalued field that contains a value that matches a full text query is considered to match the query.
 
+Full text functions are more performant for text search use cases than using pattern matching or regular expressions with `LIKE` or `RLIKE`.
```

Comment:
How about:

Full text functions are significantly more performant for text search use cases on large data sets than using pattern matching or regular expressions with `LIKE` or `RLIKE`.

Reply:
Looks good! 69fee785d276c1e4a25db66822a496987b3a0384